perf/tools/config_bench.json

```diff
@@ -0,0 +1,365 @@
+[
```

Comment:
Do we need this file included in the repository? I doubt that when someone adds a new benchmark, he will update the config.
Shouldn't it be generated for the tests if we want?
Also, I see that this file is just a human-readable version of the written config. In this case we can just drop it, I suppose.

Reply:
Yes, we need. Actually, we discussed it verbally before requesting review.
I'll describe the idea behind it here, too:
There is a single source of truth about benchmarks and their metrics - tests itself. This config is needed for preparing data for Apache Otava and the config can be generated only from benchmark results (it means that all benchmarks should be executed for generating config, and you know it is a time-consuming thing). So I am sure that config is required now. After implementing an InfluxDB importer in Otava [^1] both Python script and benchmark config will be gone.

>  I doubt that when someone adds a new benchmark, he will update the config.

Ok, I'll add a check for consistency of the bechmark config.

[^1]: https://github.com/apache/otava/issues/49